socorro/external/es/crashstorage.py

```diff
@@ -201,6 +201,7 @@ class ESCrashStorage(CrashStorageBase):
     # have been tested with Elasticsearch 1.4.
     field_name_string_error_re = re.compile(r"field=\"([\w\-.]+)\"")
     field_name_number_error_re = re.compile(r"\[failed to parse \[([\w\-.]+)]]")
+    field_name_unknown_property_error_re = field_name_number_error_re
```

Comment:
These regular expressions are so janky. But the messages are not structured, so I'm not sure whether there are better options. :(